packages/orchestration/src/examples/stake-bld.contract.js

```diff
@@ -112,7 +113,8 @@ export const start = async (zcf, privateArgs, baggage) => {
             trace('makeStakeBldInvitation', give);
             const { holder } = await makeLocalAccountKit();
             /** @type {Record<string, Payment<'nat'>>} */
-            // @ts-expect-error XXX PaymentPKeywordRecord throught deeplyFulfilled will be a PaymnentKeywordRecord
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- fails only through some build paths
+            // @ts-ignore XXX PaymentPKeywordRecord throught deeplyFulfilled will be a PaymnentKeywordRecord
```

Comment:
Let's fix the typos. This is really hard to read as is.

```suggestion
            // @ts-ignore XXX PaymentPKeywordRecord, though after deeplyFulfilled it will be a PaymentKeywordRecord
```

Reply:
if I have to do another push I'll include this

re: `b1-${string}` above, I tried that and a bunch of places have full wide `string` or use a different literal like `id` (in a test). So out of scope

Reply:
Where do these numbers come from? What do they get compared to?

Reply:
try `git grep typeCoverage`

Reply:
> try git grep typeCoverage

That pointed to the entry in `package.json`, which is where the change was that instigated my comment. 

I now see that there is a tool for measuring current type coverage. In my current PR, it tells me that **The type coverage rate(97.75%) is lower than the target(97.77%).**.   Is this something that we're expected to monitor and address?  Will it be checked in CI?  

The questions I was trying to ask before are about whether we're keeping stats over time on these metrics. When I think about test coverage (which I presume is analogous), I like to see stats over time, and be able to compare different packages. 

Has there been some discussion of a goal for this stat?

Reply:
> Is this something that we're expected to monitor and address?

It's not updated or checked in CI. It's run on a voluntary basis. I use it before a big type change to make sure I haven't lost coverage.

> Will it be checked in CI?

Updates are welcomed as of a PR. It helps the reviewers to see any deltas in coverage.

> Has there been some discussion of a goal for this stat?

Not TMK.